solutions/Hawk/solution.ini

```diff
@@ -1,6 +1,6 @@
 [build]
-default=mvn -B --quiet -f org.hawk.ttc2018/pom.xml package
-skipTests=mvn -B --quiet -f org.hawk.ttc2018/pom.xml package -DskipTests
+default=./build-hawk-neo4j.sh && mvn -B --quiet -f org.hawk.ttc2018/pom.xml package && mvn -B --quiet -f org.hawk.ttc2018/pom.xml test -Dbackend=neo4j
```

Comment:
Why is a `package` command run for Greycat, but a `test` run for Neo4j?

Reply:
See above. We don't need to recompile for Neo4j, because we actually have no compile dependencies on either Greycat or Neo4j. The solution uses reflection to create an instance of the appropriate backend implementation, so we only need to use Maven profiles to introduce the right `<dependency>` (which pulls in all the other transitive deps).

Reply:
Basically, that test run is just making sure that swapping in Neo4j works and produces the expected results.

Reply:
Oh, I see.